caravel/views.py

```diff
@@ -1304,10 +1324,12 @@ def sql_json(self):
                 sql = sql.strip().strip(';')
                 qry = (
                     select('*')
-                    .select_from(TextAsFrom(text(sql), ['*']).alias('inner_qry'))
+                    .select_from(TextAsFrom(text(sql), ['*'])
+                                 .alias('inner_qry'))
                     .limit(limit)
                 )
-                sql = str(qry.compile(eng, compile_kwargs={"literal_binds": True}))
+                sql = str(qry.compile(
```

Comment:
Mmmh, `str` isn't in the main namespace in PY3. I wonder what `type` comes out of compile and if the str is even required...